lib/rex/post/meterpreter/extensions/stdapi/railgun/def/windows/def_wldap32.rb

```diff
@@ -26,13 +26,13 @@ def self.create_library(constant_manager, library_path = 'wldap32')
     ], 'ldap_bind_sA', "cdecl")
 
     dll.add_function('ldap_search_sA', 'DWORD',[
-        ['DWORD', 'ld', 'in'],
+        ['PHANDLE', 'ld', 'in'],
         ['PCHAR', 'base', 'in'],
         ['DWORD', 'scope', 'in'],
         ['PCHAR', 'filter', 'in'],
         ['PCHAR', 'attrs[]', 'in'],
         ['DWORD', 'attrsonly', 'in'],
-        ['PDWORD', 'res', 'out']
+        ['PULONG_PTR', 'res', 'out']
```

Comment:
Tbh I didn't concern myself too much with this since they are both aliases as you mentioned but I agree syntactically this makes more sense. Will fix this up now.

Reply:
Fixed with https://github.com/rapid7/metasploit-framework/pull/17562/commits/b9c26da2afb9bd1b9b265762c9c454a79a82e379